app/src/main/java/com/trunghoang/orderhub/ui/login/LoginFragment.kt

```diff
@@ -86,6 +89,8 @@ class LoginFragment : Fragment() {
             ),
             Toast.LENGTH_SHORT
         ).show()
+        if (res.data != APIResponse.NO_VALUE)
```

Comment:
em có thể sử dụng extension ntn cho hiệu quả
https://stackoverflow.com/questions/36826004/kotlin-for-android-toast

Reply:
e vừa đẩy code mới rồi nhưng chỗ này nó ko update ạ